librato/librato.py

```diff
@@ -19,16 +19,18 @@ def __init__(self, source, options):
         # self._url = build_url(source.get("user"), source.get("token"))
         self._meta_page = 0
         self._current_metric = 0
-        self._auth = base64.encodestring("%(user)s:%(token)s" % source).strip()
+        self._auth = base64.b64encode("%(user)s:%(token)s" % source).strip()
 
         # create the list of metrics along with their start timestamp
+        source_metrics = source.get("metrics", [])
```

Comment:
You are right I thought about that too, I agree about changing the first line to `None` instead of `[]`

But on the second line it may be dangerous to change it to `is not None` since the `source.get("metrics", None)` line may produce values like `''` or `False` or some other non iterable values which are not None